hcl/ast/expression.go

```diff
@@ -394,6 +394,10 @@ func renderString(bytes []byte) []byte {
 			old: "\\n",
 			new: "\n",
 		},
+		{
+			old: `\"`,
+			new: `"`,
+		},
```

Comment:
missing `\\` `\r` and \uNNNN and \UNNNNNNNN ;)

Reply:
as we discussed in slack, I made it output quoted string if any `\r` or `\u` is present.
But fixed `\\` inside heredoc as well.